packages/longhorn/longhorn.patch

```diff
@@ -0,0 +1,366 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/Chart.yaml packages/longhorn/charts/Chart.yaml
```

Comment:
The binary files and changes in yaml files above should not be added, those will be added by the CI process.

Reply:
Now that I look at this, there are quite a few things that need to be done in order to have this qualify in the dashboard UI. I will send you a link of how to implement, but things that are missing are: 

* add custom private registry settings
* re-format image repository/tag so they get picked up for airgap
* add RBAC ClusterRoles for aggregating to default k8s clusterRoles

Reply:
1. What do you mean by `The binary files and changes in yaml files above should not be added, those will be added by the CI process.`?
2. I still have no idea about how to modify the RBAC part after reading the doc. Is there an example available?

Reply:
1. He's referring to all the changes in `index.yaml` and `assets/*` that have been included in this PR; all those files changes should be removed. This PR should only be committing the `longhorn.patch` and the `package.yaml` since the remaining files are automatically generated by CI on push.
2. This just involves adding a label to your `ClusterRoles` in order to aggregate them into the default cluster roles, mentioned in the [k8s docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles). The [labels](https://github.com/rancher/charts/blob/f53ab23985965dd79b6d09d5771d1dd23de88716/packages/rancher-logging/overlay/templates/userroles.yaml#L6) added to the `ClusterRoles` created by the `rancher-logging` chart are a great example of this.

Reply:
Part 2 we should talk about in a design discussion. The goal is that a user with an existing clusterRole of edit in the installation namespace would get whatever privileges are relevant for them.

Reply:
The changes/conflicts in `index.yaml` and `assets/*` are introduced by the automatically generated commit. I didn't touch that part.

Reply:
I just add labels in the roles in the chart for the ClusterRole `admin` and `edit` aggregation, add launch an extra role in `/overlay` for the ClusterRole `view` aggregation. Let me know if there is anything not right.

Reply:
The generated changes are happening because this is on branch named dev-v2.5, we should fix that on our side but for now can you change the branch name.